app_versions.json

```diff
@@ -1,5 +1,5 @@
 {
-    "trino": "384",
+    "trino": "424",
```

Comment:
nit: 426 is the newest

Reply:
I used 424 as it's the first one which supports `setSessionUser()`. @andrewdibiasio6 do you know if there is any downside specifying the newest?

Reply:
384 was the newest when the project was created. Now downside besides ensure it works, which we hope the tests take care of.

Reply:
Version updated